build.sbt

```diff
@@ -1,6 +1,8 @@
 import sys.process.*
 
-val scala3Version = "3.4.1"
+val scala3Version   = "3.4.1"
+val scala212Version = "2.12.19"
```

Comment:
I see no value in supporting 2.12 - there are some unexpected differences in collection behavior which may be undesirable.

Reply:
The use of collections is very limited in the scheduler. There are companies still on 2.12, it'd be nice to have the scheduler available for them as well.